src/main/kotlin/no/nav/familie/ef/sak/repository/domain/TilkjentYtelse.kt

```diff
@@ -3,15 +3,17 @@ package no.nav.familie.ef.sak.repository.domain
 import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
 import org.springframework.data.annotation.Id
 import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
 import java.time.LocalDate
 import java.util.*
 
 data class TilkjentYtelse(@Id
                           val id: UUID = UUID.randomUUID(),
-                          val behandlingId: Long, /// TODO  Gjøres om til UUID og "ekte" foreign key
+                          val behandlingId: UUID,
+                          @Column("behandling_ekstern_id")
+                          val behandlingEksternId: Long,
```

Comment:
Trenger vi eksternId når vi har behandlingsid-en? Jeg så egentlig for meg at vi kunne hatt en peker til en Behandling her, tilsvarende det vi har gjort for foreign keys med jpa, men hvis det er mye hassle så kan vi droppe det. Men da kan vi kanskje hente opp hele behandlingen et sted på veien når vi skal lage utbetalingsoppdrag, så har vi tilgang til ekstern id, fagsakId og det man ellers trenger å hente fra den.

Reply:
Samme gjelder sql lenger nede. Har du en fremednøkkel til behandling, så trenger du ikke en fremmednøkkel til behandling_ekstern også.